PKGBUILD

```
@@ -1,7 +1,7 @@
 pkgname="dollarskip"
-pkgver="3.1.2"
+pkgver="3.1.3"
 pkgrel="1"
-source=(https://github.com/CleanMachine1/DollarSkip/archive/3.1.2.tar.gz)
+source=(https://github.com/CleanMachine1/DollarSkip/archive/refs/tags/v3.1.3.tar.gz)
```

This suggested link or https://github.com/CleanMachine1/DollarSkip/archive/v3.1.3.tar.gz

the 2nd is more succinct

---

Please apply my link I have suggested unless you have anything to add.

```suggestion
source=(https://github.com/CleanMachine1/DollarSkip/archive/v3.1.3.tar.gz)
```